transfer-products/cloudformation.yml

```diff
@@ -0,0 +1,86 @@
+AWSTemplateFormatVersion: 2010-09-09
+
+Parameters:
+  HyP3URL:
+    Type: String
+
+  EarthdataUsername:
+    Type: String
+
+  EarthdataPassword:
+    Type: String
+    NoEcho: true
+
+  S3TargetPrefix:
+    Type: String
+
+Resources:
+  LogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub "/aws/lambda/${Lambda}"
+      RetentionInDays: 731
+
+  Role:
+    Type: AWS::IAM::Role
+    Properties:
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          Action: sts:AssumeRole
+          Principal:
+            Service: lambda.amazonaws.com
+          Effect: Allow
+      Policies:
+        - PolicyName: policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
```

Comment:
also need to allow `s3:PutObject` here, and maybe `s3:PutObjectTagging` (boto3's upload_file is a wrapped function and I don't quite remember which API calls it makes)

Reply:
as well as `s3:GetObject` and `s3:GetObjectTagging` if we ever switch from the transfer_object to the copy_object implementation